resources/migrations/001_update_migrations.yaml

```diff
@@ -11815,6 +11815,31 @@ databaseChangeLog:
             tableName: data_edit_undo_chain
             columnName: scope
 
+  - changeSet:
+      id: v55.2025-04-29T12:02:00
+      author: qnkhuat
+      comment: Add irreversible column
+      preConditions:
+        - not:
+            - columnExists:
+                tableName: data_edit_undo_chain
+                columnName: undoable
+      changes:
+        - addColumn:
+            tableName: data_edit_undo_chain
+            columns:
+              - column:
+                  name: undoable
+                  type: ${boolean.type}
+                  remarks: Identifies whether a change can be undo
+                  constraints:
+                    nullable: false
+                  defaultValue: true
```

Comment:
wdyt about removing the default value in the next migration, so we just use it for a once off backfill?

also, why is this not added at the end of the migrations?

Reply:
I want to group these migrations together so if we decided to rework them it's easier to find.

Reply:
> wdyt about removing the default value in the next migration

i understand why you want to do it but tbh most non-nullable boolean columns has a default value so I think it's better to be consistent here.